tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/wormhole/test_and.py

```diff
@@ -26,8 +26,11 @@ def run_and_test(input_shape, dtype, dlayout, in_mem_config, out_mem_config, dat
     if in_mem_config[1] == "SYSTEM_MEMORY":
         in_mem_config[1] = None
 
-    x = torch.Tensor(size=input_shape).uniform_(-100, 100)
-    y = torch.Tensor(size=input_shape).uniform_(-100, 100)
+    x = torch.Tensor(size=input_shape).uniform_(-100, 100).to(torch.bfloat16)
+    y = torch.Tensor(size=input_shape).uniform_(-100, 100).to(torch.bfloat16)
+
+    x = torch.where(x.abs() > 1e-3, x, 1e-3)
```

Comment:
This test still fails with low pcc even with these changes. Not sure If that is expected.